pyproject.toml

```diff
@@ -43,3 +43,27 @@ Django = "^2.2"
 [build-system]
 requires = ["poetry-core>=1.0.0a5"]
 build-backend = "poetry.core.masonry.api"
+
+
+[tool.isort]
+force_single_line = true
+force_sort_within_sections = true
+line_length = 110
+order_by_type = false
+skip_glob = [
+	"*/.tox/*",
+	"*/node_modules/*.py",
+	"*/migrations/0*.py",
```

Comment:
why 0 only? 😆

Reply:
If you have 1,000 migrations then I'm sure you have bigger (literally) issues to deal with.

That's actually copied from an old template: it's no longer in more recent projects since we just let black reformat migrations now.